api/hrlh.rb

```diff
@@ -27,23 +36,41 @@ def database
         requires :email, type: String
       end
       post do
-        database['interviewer']
-        .set(
-            params[:email],
+        interview_db
+        .create(
             {
                 name: params[:name],
                 email: params[:email]
-            },
-            false
+            }
         ).value
       end
+
+      desc 'single interviewer'
+      params do
+        requires :key, type: String
+      end
+      route_param :key do
+        get do
+          to_interviewer(interview_db[params[:key]])
+        end
+
+        put do
+          interviewer = interview_db[params[:key]]
+          interviewer[:name] = params[:name] if params[:name]
+          interviewer[:email] = params[:email] if params[:email]
+          interviewer.save
+        end
+
+        delete do
+          interview_db.delete(params[:key])
+        end
+      end
     end
 
   resource :interviewers do
     desc 'get all interviews'
     get do
-      interviewer =database['interviewer']
-      interviewer.find_all.map(&:value)
+      interview_db.map{|result| to_interviewer(result)}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.